eclair-core/src/main/scala/fr/acinq/eclair/payment/relay/Relayer.scala

```diff
@@ -234,6 +236,10 @@ object Relayer extends Logging {
       isPublic = commitments.announceChannel)
   }
   case class OutgoingChannels(channels: Seq[OutgoingChannel])
+
+  // internal classes, used for testing
+  private[payment] case class GetChildActors(replyTo: ActorRef)
+  private[payment] case class ChildActors(postRestartCleaner: ActorRef, channelRelayer: ActorRef, nodeRelayer: ActorRef)
```

Comment:
This is nice! I had to do a very ugly workaround [here](https://github.com/ACINQ/eclair/blob/2c437421d7d03435a761a76796ff438a93295617/eclair-core/src/test/scala/fr/acinq/eclair/payment/RelayerSpec.scala#L558), can you fix this too?

Reply:
Done with 3b1a4d5fa. I went one step further in another PR and added those child actors to the fixture.